router/traces_observers/summary.py

```diff
@@ -48,24 +48,59 @@ def perform(
             if settings.TRACE_SUMMARY_DELAY:
                 time.sleep(3)
 
-            prompt = f"""
-            Generate a concise, one-line summary of the trace of the action, in {language}.
-            This summary must describe the orchestrator's action, referring to all actions as "skills."
-
-            Guidelines for your response:
-            - Use the following language for the summary: {language}.
-            - The text to be summarized is the trace of the action.
-            - Use a systematic style (e.g., "Cancel Order skill activated", "Forwarding request to Reporting skill").
-            - The summary must not exceed 10 words.
-            - Use varied gerunds (e.g., "Checking", "Cancelling", "Forwarding").
-            - Do not include technical details about models, architectures, language codes, or anything unrelated to summarizing the action.
-
-            Here is the trace of the action:
-            {json.dumps(trace_data, indent=2)}
-            """
+            prompt = f"""You are an expert summarizer.
+
```

Comment:
You should remove the indentation at the beginning of each line for multi-line strings, otherwise these tabs are going to request body.